crates/uv/tests/python_pin.rs

```diff
@@ -635,26 +635,28 @@ fn python_pin_resolve() {
 fn python_pin_with_comments() -> Result<()> {
     let context = TestContext::new_with_versions(&[]);
 
-    let child = context.temp_dir.child("foo");
-    child.create_dir_all()?;
+    let content = r#"3.12
```

Comment:
```suggestion
    let content = indoc::indoc! {r"
        3.12
        # 3.11
        3.10
    "};
```

Reply:
We use r#"..."# all over the place, idk if I have a preference.

Reply:
Do you prefer the following format?

```rust
    let content = r"
3.12

# 3.11
3.10
";

```

I made that suggestion because of a clippy error, `error: unnecessary hashes around raw string literal` encountered in the CI[^1], as well as inconsistencies in indentation.

I don't have a strong preference but lean towards indentation with indoc, though.

[^1]: https://github.com/astral-sh/uv/actions/runs/10064000423/job/27820018371#step:5:54

Reply:
Sorry using the hashes as a stand-in for our general format (as in the example you shared). We usually skip using indoc to remove indentation since the tests are generally only singly-nested. I don't really have a preference though.